client/state/analytics/actions.js

```diff
@@ -0,0 +1,47 @@
+import fromPairs from 'lodash/fromPairs';
+
+import {
+	BUMP_STAT,
+	RECORD_EVENT,
+	RECORD_PAGE_VIEW
+} from './constants.js';
+
+export const withAnalytics = ( analytics, action ) =>
+	Object.assign( {}, action, { meta: Object.assign( {}, action.meta, analytics.meta ) } );
+
+export const bumpStat = ( group, name ) => ( {
+	type: BUMP_STAT,
+	meta: {
```

Comment:
I think I like `meta-analytics`. If you read the commit log for this, I noted a similar quandary. Unfortunately we can't just make `meta` an array like I'd prefer because of the way the other tooling around `meta` in Flux Standard Actions work.

The `redux-analytics` module does support a custom selector function to make it easy to choose a different meta name.